CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultActivities.uc

```diff
@@ -198,14 +241,33 @@ static function CreateCounterDarkEvent (out array<X2DataTemplate> Templates)
 
 	Activity.OnSuccess = OnSuccessPOI;
 	Activity.MissionRewards.AddItem('Reward_Rumor');
-	Activity.MissionRewards.AddItem('Reward_SmallIntel');
```

Comment:
Interesting choice for a change

Reply:
The previous infiltration mission already has a SmallIntel reward, I didn't want to have two subsequent missions with repeated rewards.

Reply:
The code is commented out

Reply:
I know but in the uncommented Assault version of this template, SmallIntel is missing there as well so my previous comment still applies.